includes/multi-currency/class-wc-payments-multi-currency-currency.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Class WC_Payments_Multi_Currency_Currency
+ *
+ * @package WooCommerce\Payments
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Multi Currency Currency object.
+ */
+class WC_Payments_Multi_Currency_Currency {
```

Comment:
I'm wondering whether we may want to use namespacing for the Multi Currency related code, so that we can just call this class `Currency`. I'm not sure if this is a practice usually followed in the plugin code (@luizreis may know more about this!) but I noticed it's being done with the [Payment Providers](https://github.com/Automattic/woocommerce-payments/blob/develop/includes/payment-methods/class-cc-payment-gateway.php).

So we could have something like:

```php
/**
 * Class WC_Payments_Multi_Currency_Currency
 *
 * @package WooCommerce\Payments
 */

namespace WCPay\Multi_Currency;

defined( 'ABSPATH' ) || exit;

/**
 * Multi Currency Currency object.
 */
class Currency {
```

Reply:
Updated to use namespaces in https://github.com/Automattic/woocommerce-payments/pull/1795/commits/cdb969523e396de384ea946505bdcbb3a4b144b9

I did run into some PHPCS issues with that, though. Will discuss in Slack.